drltrace/CMakeLists.txt

```diff
@@ -70,7 +68,12 @@ use_DynamoRIO_extension(drltracelib drwrap_static)
 use_DynamoRIO_extension(drltracelib drx_static)
 use_DynamoRIO_extension(drltracelib drsyscall_static)
 use_DynamoRIO_extension(drltracelib drcovlib_static)
+use_DynamoRIO_extension(drltracelib droption)
 
+if (UNIX)
+  # to avoid conflicts with new declaration with "C" linkage in utils.h
```

Comment:
Was droption.h fixed?  If so then this is not needed.  If not yet, then there should be a comment explaining that this is temporary.  The current comment doesn't explain the issue.